model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -721,14 +721,36 @@ message StateKey {
     bytes key = 4;
   }
```

Comment:
It would be great if we could also fill in documentation similar to MultimapSideInput/MultimapKeysSideInput at the message level for BagUserState and the two that you have added.

Reply:
lgtm